dotcom-rendering/src/web/components/ContainerLayout.tsx

```diff
@@ -32,6 +32,7 @@ type Props = {
 	ophanComponentLink?: string;
 	containerPalette?: DCRContainerPalette;
 	innerBackgroundColour?: string;
+	showLeftColumn?: boolean;
```

Comment:
Rather than extend ContainerLayout to turn off features maybe we could give thashers a simpler layout?

A bit like 

```ts
					if (isThrasher) {
						return (
							<ElementContainer
								key={index}
								padded={false}
								showSideBorders={false}
							>
								<DecideContainer
									trails={trails}
									containerType="fixed/thrasher"
									containerPalette={
										collection.containerPalette
									}
								/>
							</ElementContainer>
						);
					} else {
						return (
							<ContainerLayout
```

Arguably thrashers have too many different use cases mixed into one and we will likely need to deal with edge cases in the thrasher code itself